### ISSUE TITLE
End of Level Animations...

### ISSUE THREAD

**Issue by datahead8888** [Secretchronicles/TSC#275]
We should add even of level animations for when Alex completes a level.  They could be of objects moving in the sky, Alex walking, Alex making a gesture, or anything else.

See also: http://forum.secretchronicles.de/topics/157

**Comment by Luiji** [comment]
Personally, I don't think this is necessary. It seems fine that when Alex is done with a level he simply walks through the final door/pipe/etc. Also all three of the examples you gave are various Marioisms :) (Objects moving in sky = fireworks, Alex walking = directly + doesn't make sense in the context of how levels end, Alex making a gesture = Mario's peace sign.)

**Comment by datahead8888** [comment]
> Alex walking = directly + doesn't make sense in the context of how levels end

You are correct.  Alex cannot walk off the right side of the screen if the exit is a pipe or door.

> Also all three of the examples you gave are various Marioisms :) (Objects moving in sky = fireworks, Alex walking = directly + doesn't make sense in the context of how levels end, Alex making a gesture = Mario's peace sign.)

I understand what you are saying, but Nintendo can't have a monopoly on end of level animations.  If they're different, they might work.
* For a gesture he could shoot his fist in the air.  It's a bit different than a peace sign.
* For an object in the sky, we certainly don't want fireworks.  Some other possibilities (varying by level), are a bird flying by, an enemy air vehicle flying overhead, and an animation involving enemies on the ground.
* For door exits we could look at having an animation where Alex opens a door and scurries off behind it in the background.  For pipe exits, we could have him pause, glance into it, and then leap upward and fall into the pipe.

We might see if anyone has more thoughts on this.
